manta-pay/src/key.rs

```diff
@@ -153,26 +156,33 @@ impl<C> KeySecret<C>
 where
     C: CoinType,
 {
+    #[inline]
+    /// Gets [`Mnemonic`]
+    pub fn expose_mnemonic(&self) -> &Mnemonic {
+        &self.mnemonic
+    }
```

Comment:
Also, move this below the `new` function.

```suggestion
    /// Exposes a shared reference to the [`Mnemonic`] for `self`.
    #[inline]
    pub fn expose_mnemonic(&self) -> &Mnemonic {
        &self.mnemonic
    }
```